src/consts/manage-app-secret.ts

```diff
@@ -0,0 +1,5 @@
+export enum APP_SECRET_MANAGEMENT_MODES {
+  LIST_KEYS = 'list-keys',
+  SET = 'set',
+  DELETE = 'delete',
+}
```

Comment:
this is identical to what we have in "Env" - is there any reason to duplicate this file and contents, instead of re-using what we already have?

Reply:
I though that it will be the best to separate between those two action, because maybe in the future we would like to change some management mode only for one of them and not for the other. Let me know if you still think this duplication isn't necessary.

Reply:
I understand, but this goes for everything really- but for types, interfaces, enums, consts and things that are "general" (i.e not business logic code) - we try to re-use what we can.
If we need to in the future, we can expand on that.